pkg/controller/kibana/driver.go

```diff
@@ -219,7 +221,18 @@ func (d *driver) Reconcile(
 		return results.WithError(err)
 	}
 
-	results.WithResults(kbcerts.Reconcile(ctx, d, *kb, []corev1.Service{*svc}, params.CACertRotation, params.CertRotation))
+	_, results = certificates.ReconcileCAAndHTTPCerts(
```

Comment:
Again, we can probably wrap this in an `if` clause and invoke it only if TLS is enabled.